pro/src/screens/Offers/__specs__/Offers.spec.tsx

```diff
@@ -349,89 +322,20 @@ describe('screen Offers', () => {
-      it('should disable status filters when no venue nor offerer filter is selected', () => {
-        renderOffers(props)
-
-        expect(
-          screen.getByRole('button', {
-            name: 'Statut Afficher ou masquer le filtre par statut',
-          })
-        ).toBeDisabled()
-      })
-
-      it('should disable status filters when no venue filter is selected, even if one venue filter is currently applied', async () => {
-        renderOffers({
-          ...props,
-          initialSearchFilters: {
-            ...DEFAULT_SEARCH_FILTERS,
-            venueId: 'JI',
-          },
-        })
-
-        await userEvent.selectOptions(
-          screen.getByDisplayValue('Ma venue'),
-          'all'
-        )
-
-        expect(
-          screen.getByRole('button', {
-            name: 'Statut Afficher ou masquer le filtre par statut',
-          })
-        ).toBeDisabled()
-      })
-
-      it('should enable status filters when venue is selected but filter is not applied', async () => {
-        renderOffers(props)
-        const venueOptionToSelect = screen.getByRole('option', {
-          name: proVenues[0].name,
-        })
-
-        await userEvent.selectOptions(
-          screen.getByLabelText('Lieu'),
-          venueOptionToSelect
-        )
-
-        expect(
-          screen.getByRole('button', {
-            name: 'Statut Afficher ou masquer le filtre par statut',
-          })
-        ).not.toBeDisabled()
-      })
-    })
-
     describe('select all offers checkbox', () => {
-      it('should disable select all checkbox when no venue nor offerer filter is applied', () => {
-        renderOffers(props, { user: currentUser })
+      it('should disable select all checkbox', () => {
+        renderOffers({ ...props, canHaveAStatusFilter: false })
 
         const selectAllOffersCheckbox =
           screen.getByLabelText('Tout sélectionner')
         expect(selectAllOffersCheckbox).toBeDisabled()
       })
 
-      it('should disable select all checkbox when venue filter is not set', async () => {
-        renderOffers(props, { user: currentUser })
-
-        await userEvent.selectOptions(screen.getByLabelText('Lieu'), 'all')
-
-        await userEvent.click(screen.getByText('Rechercher'))
-
-        const selectAllOffersCheckbox =
-          screen.getByLabelText('Tout sélectionner')
-        expect(selectAllOffersCheckbox).toBeDisabled()
-      })
-
-      it('should enable select all checkbox when venue filter is applied', async () => {
-        renderOffers(
-          {
-            ...props,
-            initialSearchFilters: {
-              ...DEFAULT_SEARCH_FILTERS,
-              venueId: 'IJ',
-            },
-          },
-          { user: currentUser }
-        )
+      it('should enable select all checkbox', async () => {
+        renderOffers({
+          ...props,
+          canHaveAStatusFilter: true,
```

Comment:
Peut-être expliquer un peu plus pourquoi on peut ou non sélectionner toutes les offres :
... when performances not allow it ?
ou
... when user can filter on status ?
ou autre

Reply:
Carrément, j'ai revu le nommage de ces tests 👍